python/cuvs/cuvs/neighbors/cagra/cagra.pxd

```diff
@@ -49,13 +62,20 @@ cdef extern from "cuvs/neighbors/cagra.h" nogil:
 
     ctypedef cuvsCagraCompressionParams* cuvsCagraCompressionParams_t
 
+    ctypedef struct cuvsIvfPqParams:
```

Comment:
I know that this PR didn't introduce this struct - but I think that we should rename this to `cuvsCagraIvfPqParams` since this is for cagra only (and could be confusing for users of ivf-pq). @lowener can you create an issue to change this for 25.08? I think its a bit late to try to get this in for 25.06

Reply:
The all neighbor API recently introduced it's own version of this struct: https://github.com/jinsolp/cuvs/blob/branch-25.06/cpp/include/cuvs/neighbors/all_neighbors.hpp#L37.
I think we can keep the name and factorize both struct into `ivf_pq.hpp`